docs/main_viewer.md

```diff
@@ -0,0 +1,82 @@
+# Main Viewer
+
+![](./pngs/blank_main_viewer.png)
+
+When openend, AMP will display its main image viewer.  This window will contain figures generated
+by plugins as well as direct images loaded in via the 'Open Cohort' menu option.
+
+### Cohort Loading
+
+To load a cohort of images into the viewer, find the `Open Cohort...` option under the `File`
+section of the upper taskbar.
+
+![](./pngs/open_cohort.png)
+
+This will open a file explorer.  Navigate to the folder containing your images, select it, and 
+click the `Open` button.
+
+<div 
+    style="
+        border: 0px solid #35f;
+        border-left-width: 6px;
+        padding: 8px;
+        margin-bottom: 8px;
+        background-color: #acf;
+        color: #000"
+> 
+    <b><i>Note</i></b>:  Cohorts can be loaded from any file depth less than 6
+</div>
+
+![](./pngs/cohort_file_explorer.png)
+
+If the cohort was successfully loaded, you should now see a file tree within the `Files` section of
```

Comment:
I think it would be useful to put a big **1** next to files on the screenshot, and a big **2** next to Figures on the screenshot. Then in the docs, you can refer to the number, in addition to the name, to guide people for where to look